modules/javafx.controls/src/main/java/javafx/scene/control/Skin.java

```diff
@@ -60,6 +73,14 @@
      */
     public Node getNode();
 
+    /**
+     * Called by {@link Skinnable#setSkin(Skin)} on a pristine control, or after the
+     * previous skin has been uninstalled via its {@link #dispose()} method.
```

Comment:
Suggestion: You might be able to simplify this a bit by saying: "Called by {@link Skinnable#setSkin(Skin)}, after the previous skin, if any, has been uninstalled." I don't have a strong opinion on this.

Reply:
thank you